README.md

```diff
@@ -111,20 +111,20 @@ linelist <- sim_linelist(
   onset_to_death = onset_to_death
 )
 head(linelist)
-#>   id        case_name case_type gender age date_onset date_admission date_death
-#> 1  1 Hamdi el-Abdalla confirmed      m  76 2023-01-01           <NA>       <NA>
-#> 2  2     Tosha Aragon suspected      f  50 2023-01-02           <NA>       <NA>
-#> 3  3         May Chen  probable      f   7 2023-01-03     2023-01-14       <NA>
-#> 4  4    Ellen Tremmel confirmed      f  25 2023-01-02           <NA>       <NA>
-#> 5  5  Nathaniel Beard suspected      m  38 2023-01-03           <NA>       <NA>
-#> 6  6     Amanda Silas  probable      f  66 2023-01-03           <NA>       <NA>
-#>   date_first_contact date_last_contact ct_value
-#> 1               <NA>              <NA>     23.4
-#> 2         2023-01-01        2023-01-05       NA
-#> 3         2023-01-02        2023-01-04       NA
-#> 4         2023-01-02        2023-01-04     23.4
-#> 5         2023-01-01        2023-01-03       NA
-#> 6         2023-01-04        2023-01-06       NA
+#>   id              case_name case_type gender age date_onset date_admission
+#> 1  1            Sable Mills  probable      f  25 2023-01-01           <NA>
+#> 2  2        Dallin Boulware  probable      m   6 2023-01-04           <NA>
+#> 3  3            Luke Lovato  probable      m  69 2023-01-04           <NA>
+#> 4  4              Andrew Ho confirmed      m  72 2023-01-05           <NA>
+#> 5  5         Deanna Wallace suspected      f   7 2023-01-05           <NA>
+#> 6  6 Abdur Rasheed el-Aboud suspected      m  18 2023-01-05           <NA>
+#>   date_death date_first_contact date_last_contact ct_value
+#> 1       <NA>               <NA>              <NA>       NA
+#> 2       <NA>         2023-01-04        2023-01-04       NA
+#> 3       <NA>         2023-01-03        2023-01-04       NA
+#> 4       <NA>         2023-01-01        2023-01-05     22.7
+#> 5       <NA>         2023-01-09        2023-01-12       NA
+#> 6       <NA>         2023-01-06        2023-01-07       NA
```

Comment:
@joshwlambert can we set the seed in this README to avoid unnecessary diffs each time the README is re-rendered?

Reply:
Yes, I don't see a reason why we cannot set a seed. Please let me know if you're still working on this PR, if not I'll commit the changes and merge.

Reply:
I'm good with this version. I'm simply wondering if we should keep the URL in the citation or if the DOI is enough :thinking:

Reply:
Maybe we can default to current year if the `Date` field does not exist?

Reply:
I've commented below on a possible solution.

Reply:
Seed issue logged in #57.